packages/dbml-core/src/parse/ANTLR/parsers/postgresql/PostgreSQLParserBase.js

```diff
@@ -0,0 +1,14 @@
+import antlr4 from 'antlr4';
+
+// Original (C#): https://github.com/antlr/grammars-v4/blob/master/sql/postgresql/CSharp/PostgreSQLLexerBase.cs
+// Base class required by the generated parser
+export default class PostgreSQLParserBase extends antlr4.Parser {
+  constructor (input) {
+    super(input);
+  }
```

Comment:
eslint error: `Useless constructor`

Reply:
Fixed in https://github.com/holistics/dbml/pull/416/commits/5aad86bc89dac9fa4fa9180630cea993815d1b16

Reply:
I removed the constructor in https://github.com/holistics/dbml/pull/416/commits/5f5183bb523e5260d4efe04a61e27aaf66bdfde7